week2/index.js

```diff
@@ -1,65 +1,150 @@
+// require filesystem library because we are going to write and read files
 var fs = require('fs');
 
-var options = process.argv.slice(2);
+//this object will contain the todo's info and will be written to the todosFile 
+// I have chosen this to be a JSON because we can use the JSON proberities on it
+let todosFile = JSON.parse(fs.readFileSync('todo.json','UTF-8'));
+console.log(todosFile);
 
-var command = options[0];
+//shorten the arguments array to the arguments we need
+//take out the first two arguments which are the node path and the file path
```

Comment:
This is a great example of a "WHY" comment.

"Take out the first two arguments" is a WHAT.
"We don't care about the node path and the file path" is a WHY.

Keep writing comments like these 👍